lib/collection.js

```diff
@@ -65,6 +64,8 @@ const updateDocuments = require('./operations/collection_ops').updateDocuments;
 const updateMany = require('./operations/collection_ops').updateMany;
 const updateOne = require('./operations/collection_ops').updateOne;
 
+const DEFAULT_VALIDATION = 'error';
```

Comment:
I'm temporarily setting this to `error` so we can catch any test failures. I'll change it back before merging.